Moose Development/Moose/AI/AI_A2A_Dispatcher.lua

```diff
@@ -3459,10 +3468,10 @@ do -- AI_A2A_DISPATCHER
                     local DefenderTarget = Dispatcher:GetDefenderTaskTarget( DefenderGroup )
 
                     if DefenderTarget then
-                      if Squadron.Language == "EN" then
+                      if Squadron.Language == "EN" and self.SetSendPlayerMessages then
                         Dispatcher:MessageToPlayers( Squadron,  DefenderName .. " wheels up.", DefenderGroup )
-                      elseif Squadron.Language == "RU" then
-                        Dispatcher:MessageToPlayers( Squadron,  DefenderName .. " колеса вверх.", DefenderGroup )
+                      elseif Squadron.Language == "RU" and self.SetSendPlayerMessages then
+                        Dispatcher:MessageToPlayers( Squadron,  DefenderName .. " ÐºÐ¾Ð»ÐµÑ�Ð° Ð²Ð²ÐµÑ€Ñ….", DefenderGroup )
                       end
```

Comment:
Aaaarg. Garbled up the Russian messages, ignore the changes in the message itself